src/main/java/by/itechart/api/entity/User.java

```diff
@@ -0,0 +1,39 @@
+package by.itechart.api.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
```

Comment:
Not calling to change but just sharing the information: the `@Data` [annotation](https://projectlombok.org/features/Data) of Lombok includes both `@Getter` and `@Setter`

Reply:
Thank's!